App.tsx

```diff
@@ -38,6 +40,7 @@ const App: React.FC = () => (
           name={Routes.STATION_SELECT}
```

Comment:
GitHub is annoying and won't let me comment on lines that are far from the changed line, but are you using the details screen now? would be good to delete it and everything it uses if not to help keep the codebase nice and tidy